Build/Common.proj

```diff
@@ -73,7 +72,7 @@
   </PropertyGroup>
 
   <!-- References -->
-  <Import Project="$(DlrBuildDir)\$(ReferencedPlatform).props" />
```

Comment:
With the new csproj files TargetFramework should always be set. I'm not exactly sure how it behaves when you mix with old style csproj files...